lib/instana/instrumentation/grpc.rb

```diff
@@ -0,0 +1,84 @@
+call_types = [:request_response, :client_streamer, :server_streamer, :bidi_streamer]
+
+if defined?(GRPC::ActiveCall) && ::Instana.config[:'grpc'][:enabled]
+  call_types.each do |call_type|
+    GRPC::ClientStub.class_eval <<-RUBY, __FILE__, __LINE__ + 1
+      def #{call_type}_with_instana(method, *others, **options)
+        kvs = { rpc: {} }
+
+        unless ::Instana.tracer.tracing?
+          return #{call_type}_without_instana(method, *others, **options)
+        end
+
+        kvs[:rpc][:flavor] = :grpc
+        kvs[:rpc][:host] = @host
+        kvs[:rpc][:call] = method
+        kvs[:rpc][:call_type] = :#{call_type}
+
+        ::Instana.tracer.log_entry(:'rpc-client', kvs)
+
+        context = ::Instana.tracer.context
+        if context
+          options[:metadata] = (options[:metadata] || {}).merge(
+            'x-instana-t' => context.trace_id_header,
+            'x-instana-s' => context.span_id_header
+          )
+        end
+
+        #{call_type}_without_instana(method, *others, **options)
```

Comment:
This is a bug from github rendering. This line is in the `class_eval` scope and call_type is interpolated into the code snippet for evaluation.